app/utils/mappers/IndividualBeneficiaryMapper.scala

```diff
@@ -37,17 +37,19 @@ class IndividualBeneficiaryMapper {
           readIdentification and
           readAddress and
           VPE1FormYesNoPage.path.read[Boolean] and
+          RoleInCompanyPage.path.readNullable[RoleInCompany] and
           readIncome and
           IncomeDiscretionYesNoPage.path.read[Boolean] and
           StartDatePage.path.read[LocalDate]
         ) (
-        (name, dateOfBirth, nationalInsuranceNumber, address, vulnerableYesNo, income, incomeDiscretion, entityStart) =>
+        (name, dateOfBirth, nationalInsuranceNumber, address, vulnerableYesNo, roleInCompany, income, incomeDiscretion, entityStart) =>
```

Comment:
Think this can be neatened up by using `Reads(_ => JsSuccess(true))` again for provisional